feature-configs/rhacm-manifests/operator-install/dpdk/dpdk-build-install.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: dpdk-build-install
+  namespace: dpdk
+spec:
+  componentKinds:
+  - group: apps.open-cluster-management.io
+    kind: Subscription
+  descriptor: {}
+  selector:
+    matchExpressions:
+    - key: app
+      operator: In
+      values:
+      - dpdk-build-install
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Subscription
+metadata:
+  name: dpdk-subscription-4.6
+  namespace: dpdk
+  labels:
+    app: dpdk-build-install
+  annotations:
+    apps.open-cluster-management.io/git-path: feature-configs/deploy/dpdk
+    apps.open-cluster-management.io/git-branch: release-4.6
```

Comment:
let's do master and then port back if we want
which also imply let's focus here on u/s versions

Reply:
The point of doing master is that the maintained deploy folder in this branch points to a 4.7 snapshot index of the performance addon operator, see the [catalogSource](https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/deploy/performance/operator_catalogsource.yaml) while PTP or SRIOV are still targeting the official 4.6 release.

Currently, RHACM runs on OpenShift 4.6, so I do not know if it is possible to run PAO 4.7 on an OpenShift cluster 4.6 .

What we can do is move the PR to the master branch, but RHACM Git/Subscription manifests still pointing to the release-4.6 branch of deploy/ folder as it is right now and as it has been verified.

Reply:
Ah, just noticed this points to release-4.6
What we normally do, is to have master pointing to the last working version until the "right version" is available.
This is the reason why the sriov-operator installed is still 4.6. I'd second @yuvalk approach of pushing to master and backporting. If we don't do that, I am afraid of missing pieces in the process.